### PR TITLE
CDAP-2918: Fix Hive 0.12 CDH support

### PR DESCRIPTION
**Pull request by jwang47**
http://builds.cask.co/browse/CDAP-RBT411

### CODE REVIEW

File: cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java [hunk 1]
```diff
@@ -64,9 +63,11 @@
    * Hive support enum.
    */
   public enum HiveSupport {
-    HIVE_CDH4(Pattern.compile("^.*cdh4\\..*$"), HiveCDH4ExploreService.class),
-    HIVE_CDH5_4(Pattern.compile("^.*cdh5.4\\..*$"), Hive14ExploreService.class),
-    HIVE_CDH5(Pattern.compile("^.*cdh5\\..*$"), HiveCDH5ExploreService.class),
+    // CDH 5.0 to 5.1 uses Hive 0.12
+    // CDH >5.3 uses Hive >0.13.1 (aka 1.0, which Hive14ExploreService supports)
```
Comment (albertshau) [inline cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java]: does this include 5.2? or is there no 5.2?

**Comment by albertshau** [comment]
one question, otherwise lgtm assuming its been tested on relevant clusters

File: cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java (hunk 1 above)
Comment (jwang47) [inline cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java]: Typo. Should be CDH >=5.2.

**Comment by albertshau** [comment]
:+1:

**Pull request merged**